files/default.conf

```diff
@@ -40,6 +40,12 @@ server {
     root /var/www/html;
     index index.html;
 
+    add_header Content-Security-Policy "default-src *; style-src 'self' 'unsafe-inline'";
+    add_header Strict-Transport-Security 'max-age=6307200; includeSubDomains; preload';
+    add_header X-Content-Type-Options nosniff;
+    add_header X-Frame-Options "SAMEORIGIN";
+    add_header X-XSS-Protection "1; mode=block";
```

Comment:
Sorry if Mozilla is giving conflicting recommendations :flushed: but let's not include `X-XSS-Protection` since it's not supported by modern browsers and not recommended by MDN: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection